iOS/Multiplatform Redux Sample/Multiplatform Redux Sample/Main/CollectionPointMapViewController.swift

```diff
@@ -116,6 +172,13 @@ class CollectionPointMapViewController: BasePresenterViewController<CollectionPo
     private func didTapLocationButton() {
         kermit().d("Locate me!")
     }
+
+    @objc private func didTabFilterChip(_ chip: MDCChipView) {
+        var filter = self.filter
+        let changedFilter = filter[chip.tag]
+        filter[chip.tag] = MapFilterItem(collectionPointType: changedFilter.collectionPointType, isSelected: !chip.isSelected)
+        _ = dispatch(UpdateFilterAction(newFilter: filter))
+    }
```

Comment:
Hier sollte `filter` nicht verändert werden. Sondern nur die Action dispatched und die Filter-Views dann anhand des neuen ViewStates entsprechend gerendert werden.

Reply:
Das ist schon was passiert: Ich kopiere den aktuellen filter in eine lokale Variable und passe dann den geänderten Filter in der lokalen Variable an. Das Feld wird erst im `render` call geupdatet.

Reply:
Ahhhh, das verwirrende war wohl für mich, dass alle Filter geschickt werden und nicht nur welcher getapped wurde. Darum hatte ich gerade etwas Mühe... Schauen wir es nachher rasch im Daily an.